src/qml/main.qml

```diff
@@ -66,8 +73,35 @@ CutieWindow {
                 icon.height: width / 2
                 icon.width: height / 2
                 background: null
-                onClicked:
-                    compositor.execApp(model["Desktop Entry/Exec"]);
+
+                onPressed: {
```

Comment:
You should probably use [onPressAndHold](https://doc.qt.io/qt-6/qml-qtquick-controls-abstractbutton.html#pressAndHold-signal) here. IIRC, onClicked would not be called in that case upon long presses.

Reply:
So I can drop the timer?

Reply:
Yes.